### PR TITLE
Move KeyboardLayout methods to Interop User32

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Move KeyboardLayout methods to Interop User32 and update their usages.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2266)

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.GetKeyboardLayout.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [DllImport(Libraries.User32, ExactSpelling = true, CharSet = CharSet.Auto)]
+        public static extern int GetKeyboardLayoutList(int size, [Out, MarshalAs(UnmanagedType.LPArray)] IntPtr[] hkls);
```
Comment (RussKie) [inline src/Common/src/Interop/User32/Interop.GetKeyboardLayout.cs]: I think this should be
```suggestion
        public unsafe static extern int GetKeyboardLayoutList(int size, IntPtr* hkls);
```

File: src/Common/src/Interop/User32/Interop.ActivateKeyboardLayout.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [DllImport(Libraries.User32, ExactSpelling = true, CharSet = CharSet.Auto)]
+        public static extern IntPtr ActivateKeyboardLayout(HandleRef hkl, int uFlags);
```
[the remark this answers is not in the document]
Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.ActivateKeyboardLayout.cs]: Done.

File: src/Common/src/Interop/User32/Interop.GetKeyboardLayout.cs (hunk 1 above)
Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.GetKeyboardLayout.cs]: Done.

**Comment by RussKie** [comment]
Thank you

**Pull request merged**